### ISSUE TITLE
ENH: Use `input` and `target` in hedging loss

### ISSUE THREAD

**Issue by simaki** [pfnet-research/pfhedge#568]
so that it is compatible with builtin loss functions in pytorch

**Comment by simaki** [comment]
resolved
